officers/three_of_ten.json

```diff
@@ -0,0 +1,82 @@
+{
+  "description": "Once considered one of the most intelligent and beautiful members of Vulcan society, much of that has been washed away by the Borg's assimilation process, leaving T'Pring a husk of her former self. Now deployed as a ruthless and efficient drone, Three of Ten is used to help expand the Borg's range of incluence and turmoil. Whatever existed of this matriarch to a happy family has become another solider in the Borg's ongoing mission to control the galaxy.",
+  "ability_info": {
+    "captain_maneuver": {
+      "name": "Efficient Gas Acquisition",
+      "bonus": "Three of Ten increases Gas mining speed by {value}",
+      "percentage": true,
+      "value": 0.6
+    },
+    "officer_ability": {
+      "name": "Resource Protection Protocols",
+      "bonus": "Three of Ten increases Protected Cargo by {value}",
+      "percentage": true
+    }
+  },
+  "class": "science",
+  "faction": "neutral",
+  "group": "Unimatrix Twelve",
+  "img_url": "./assets/three_of_ten.png",
+  "officer_name": "Three of Ten",
+  "ranks": [
+    {
+      "name": "Ensign I",
+      "officer_ability_value": 1,
```

Comment:
Officer abilities should be 3.0, 3.5, 4.0, 4.5, 5.0

Reply:
Fixed in 927a27